exp-player/addon/components/exp-card-sort/template.hbs

```diff
@@ -78,13 +78,14 @@
                         {{#each group.categories as |category|}}
                             <div class="col-sm-4 small-bucket
                                 {{unless (eq category.cards.length category.max) 'text-danger validation-error'}}
-                                ">
-                                <h5 class="bucket-label visible-xs {{if
+                                {{if (eq category.cards.length category.max) 'text-success'}}
```

Comment:
Minor: the `if` condition is the same as the `unless` condition.  This could be combined into a single if statement with a true value and a false value.